include/boost/geometry/geometries/adapted/std_array.hpp

```diff
@@ -0,0 +1,114 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2010 Alfredo Correa
```

Comment:
This code is basically copied from the `boost::array` adaptor and that's why I did not change it. I don't think my minimal changes are above a threshold of creativity to substantiate any copyright. But I will add my name to this file if you consider it necessary.